### DISCUSSION TITLE
Help - Library error

### DISCUSSION THREAD

**Discussion by RiSi42** [mobizt/ESP-Google-Sheet-Client#24] [Q&A]
Hi everyone

I'm inexperienced and I'm trying my hand at this project, I need to add a row at the bottom of a gogole worksheet with 3 values.

I am using an RP2040 but the ESP_Google_Sheet_Client.h library seems to be incompatible with this Arduino model, it is not present in the list of compatible libraries and if I select "show incompatible" it is shown.
I'm using the Arduino Cloud online editor because that's convenient for other functions.

All I have to do is insert in the sketch: #include "ESP_Google_Sheet_Client.h" and the compilation gives me an infinite series of errors.

How can I solve this problem?
Below is the error log:

`/usr/local/bin/arduino-cli compile --fqbn arduino:mbed_nano:nanorp2040connect --build-cache-path /tmp --output-dir /tmp/1281678703/build --build-path /tmp/arduino-build-A7FB381A13748A00FF3FF1C8DADF3E55 /tmp/1281678703/Cubino_dec31a

/tmp/arduino-build-A7FB381A13748A00FF3FF1C8DADF3E55/libraries/esp_google_sheet_client_1_4_3/client/SSLClient/bssl/objs.a(aes_big_cbcdec.c.o): In function `br_aes_big_cbcdec_init':

/home/builder/Arduino/libraries/esp_google_sheet_client_1_4_3/src/client/SSLClient/bssl/aes_big_cbcdec.c:35: multiple definition of `br_aes_big_cbcdec_init'

/tmp/arduino-build-A7FB381A13748A00FF3FF1C8DADF3E55/libraries/arduinoiotcloud_1_13_0/tls/bearssl/objs.a(aes_big_cbcdec.c.o):/home/builder/Arduino/libraries/arduinoiotcloud_1_13_0/src/tls/bearssl/aes_big_cbcdec.c:35: first defined here

/tmp/arduino-build-A7FB381A13748A00FF3FF1C8DADF3E55/libraries/esp_google_sheet_client_1_4_3/client/SSLClient/bssl/objs.a(aes_big_cbcdec.c.o): In function `br_aes_big_cbcdec_run':

/home/builder/Arduino/libraries/esp_google_sheet_client_1_4_3/src/client/SSLClient/bssl/aes_big_cbcdec.c:43: multiple definition of `br_aes_big_cbcdec_run'

/tmp/arduino-build-A7FB381A13748A00FF3FF1C8DADF3E55/libraries/arduinoiotcloud_1_13_0/tls/bearssl/objs.a(aes_big_cbcdec.c.o):/home/builder/Arduino/libraries/arduinoiotcloud_1_13_0/src/tls/bearssl/aes_big_cbcdec.c:43: first defined here

/tmp/arduino-build-A7FB381A13748A00FF3FF1C8DADF3E55/libraries/esp_google_sheet_client_1_4_3/client/SSLClient/bssl/objs.a(aes_big_cbcdec.c.o):(.rodata.br_aes_big_cbcdec_vtable+0x0): multiple definition of `br_aes_big_cbcdec_vtable'

/tmp/arduino-build-A7FB381A13748A00FF3FF1C8DADF3E55/libraries/arduinoiotcloud_1_13_0/tls/bearssl/objs.a(aes_big_cbcdec.c.o):(.rodata.br_aes_big_cbcdec_vtable+0x0): first defined here

................................................................................................many other error lines

/tmp/arduino-build-A7FB381A13748A00FF3FF1C8DADF3E55/libraries/arduinoiotcloud_1_13_0/tls/bearssl/objs.a(x509_minimal.c.o):(.rodata.br_x509_minimal_vtable+0x0): first defined here

/tmp/arduino-build-A7FB381A13748A00FF3FF1C8DADF3E55/libraries/esp_google_sheet_client_1_4_3/client/SSLClient/bssl/objs.a(x509_minimal_full.c.o): In function `br_x509_minimal_init_full':

/home/builder/Arduino/libraries/esp_google_sheet_client_1_4_3/src/client/SSLClient/bssl/x509_minimal_full.c:34: multiple definition of `br_x509_minimal_init_full'

/tmp/arduino-build-A7FB381A13748A00FF3FF1C8DADF3E55/libraries/arduinoiotcloud_1_13_0/tls/bearssl/objs.a(x509_minimal_full.c.o):/home/builder/Arduino/libraries/arduinoiotcloud_1_13_0/src/tls/bearssl/x509_minimal_full.c:34: first defined here

collect2: error: ld returned 1 exit status

Multiple libraries were found for "ArduinoECCX08.h"

Used: /home/builder/opt/libraries/arduinoeccx08_1_3_7

Not used: /home/builder/opt/libraries/rak5814_atecc608a_1_0_0

Multiple libraries were found for "Wire.h"

Used: /home/builder/.arduino15/packages/arduino/hardware/mbed_nano/4.0.10/libraries/Wire

Not used: /home/builder/opt/libraries/flexwire_1_1_2

Multiple libraries were found for "SPI.h"

Used: /home/builder/.arduino15/packages/arduino/hardware/mbed_nano/4.0.10/libraries/SPI

Not used: /home/builder/opt/libraries/eventethernet_1_0_0

Multiple libraries were found for "WiFiNINA.h"

Used: /home/builder/opt/libraries/wifinina_1_8_14

Not used: /home/builder/opt/libraries/vega_wifinina_1_0_1

Error during build: exit status 1`

**Comment by mobizt** [comment]
This library supports Arduino Pico SDK and not support ArduinoCore-mbed.
https://github.com/mobizt/ESP-Google-Sheet-Client?tab=readme-ov-file#rp2040-arduino-sdk-installation
